sycl/test-e2e/Matrix/Legacy/XMX8/element_wise_all_ops_int8.cpp

```diff
@@ -6,6 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 // REQUIRES: matrix-xmx8
+// XFAIL:*
```

Comment:
Does it help?
You said in the tracker that tests are timing out, so probably it's better to skip running them at all with:

// REQUIRES: TEMPORARY_DISBLED

Reply:
Yes, these are passing now. https://github.com/intel/llvm/actions/runs/6513111676/job/17692457598?pr=11521

This is similar to how others tests are disabled. 

> Unsupported Tests (112):
>   SYCL :: Matrix/Legacy/XMX8/element_wise_all_ops_int8.cpp
> ...

Reply:
> This is similar to how others tests are disabled.

What tests do you refer to?
NOTE: they might be disable for a different reason - most likely they fail execution within the time limit. Using XFAIL in this case is reasonable way to fix the return status.

> 
> > Unsupported Tests (112):
> > SYCL :: Matrix/Legacy/XMX8/element_wise_all_ops_int8.cpp
> > ...

> Yes, these are passing now. https://github.com/intel/llvm/actions/runs/6513111676/job/17692457598?pr=11521

According to my understanding using XFAIL can't move the test to "Unsupported Tests" category. I think what plays role here is: "REQUIRES: matrix-xmx8". The link your provided is for the run on GPU device only and my guess is that none of GHA CI systems have this feature. 

Here is the link where tests are executed on CPU as well: https://github.com/intel/llvm/actions/runs/6513111676/job/17692457598?pr=11521

NOTE: that GHA CI machines are different. I think we were lucky, and tests were executed on the machine w/o matrix support, so tests skipped. 

From https://www.llvm.org/docs/TestingGuide.html#tips-for-writing-constraints

> ``UNSUPPORTED`` and ``XFAIL``

> Both of these indicate that the test isn’t expected to work; however, they have different effects. **UNSUPPORTED causes the test to be skipped; this saves execution time**, but then you’ll never know whether the test actually would start working. Conversely, **XFAIL actually runs the test but expects a failure output, taking extra execution time** but alerting you if/when the test begins to behave correctly (an XPASS test result). You need to decide which is more appropriate in each case.

I suggest replacing `XFAIL:*` with `REQUIRES: TEMPORARY_DISBLED` to disable the test on all machines.

Reply:
@aelovikov-intel, @YuriPlyakhin, can you confirm that GHA CI have machines with and w/o `matrix-xmx8` feature support and E2E tests can be randomly scheduled on any of these machines?

Reply:
@bader, GHA CI has 1-2 systems, which can run matrix-xmx8 tests, and matrix-xmx8 tests are scheduled (or will be scheduled) only on these machines. matrix-xmx8 tests are not running on other machines in GHA CI.

Reply:
Talked to @aelovikov-intel and he told me that pre-commit runs tests only in multi-device environment, so `XFAIL:*` effectively disables these tests, but we run on a single device in post-commit, where will run executed.

Reply:
Thanks @bader @aelovikov-intel and @YuriPlyakhin . Updated to use `REQUIRES: TEMPORARY_DISBLED` instead.